docs/project/release-process.md

```diff
@@ -4,60 +4,27 @@
---max-issues 1 \
--o
-```
-
-1. Review each change log item.
-   * Make sure that sentences are grammatically correct and well formatted (although we will try to enforce this at the PR review stage). 
-   * Make sure that each item is categorised correctly. You will see the following categories: `Breaking changes`, `Implemented enhancements`, `Fixed bugs`, and `Merged pull requests`. Any unlabelled PRs will be found in `Merged pull requests`. It's important to make sure that any `breaking changes`, `enhancements`, or `bug fixes` are pulled up out of `merged pull requests` into the correct category. Housekeeping, tech debt clearing, infra changes, or refactoring do not count as `enhancements`. Only enhancements a user benefits from should be listed in that category.
-   * Make sure that the "Full Change log" link is actually comparing the correct tags (normally your released version against the previously version).
-   * Make sure that release notes and breaking changes are present.
-
-### Flag Breaking Changes & Deprecations
-
-It's important to flag breaking changes and deprecation to the API for each release so that we can maintain API compatibility.
-
-Developers should have flagged PRs with breaking changes with the `compat/breaking` label. However, it's important to double check each PR's release notes and contents for changes that will break API compatibility and manually label `compat/breaking` to PRs with undeclared breaking changes. The change log will have to be regenerated if any new labels have to be added.
+### Pre-release Verification (Verification that wheels are built correctly) for minor release.
+1. Merge upstream master changes into your **fork**. Make sure you are running the workflow off of your fork!
+2. Create a tag manually for the release on your fork. For example, if you are doing a release for version 0.22.0, create a tag by doing the following.
+   - Checkout master branch and run `git tag v0.22.0`.
+   - Run `git push --tags` to push the tag to remote.
+3. Access the `Actions` tab on your github UI on your fork and click the `build_wheels` action.
+4. Look for the header `This workflow has a workflow_dispatch event trigger` and click `Run Workflow` on the right.
+5. Run the workflow off of the tag you just created(`v0.22.0` in this case) and verify that the workflow worked.
+
+### Pre-release Verification (Verification that wheels are built correctly) for patch release.
+1. Check out the branch of your release (e.g `v0.22-branch` on your local **fork**) and push this to your fork (`git push -u origin <branch>`).
+2. Cherry pick commits that are relevant to the patch release onto your forked branch.
+3. Checkout the release branch and add a patch release tag (e.g `v0.22.1`) by running `git tag <tag>`.
+4. Push tags to your origin branch with `git push origin <tag>`.
+5. Kick off `build_wheels` workflow in the same way as is detailed in the last section on of the patch release tag.
+
+### Release
```

Comment:
What stuff does this workflow do? Python release? Java release? UI release? Helm charts? Docker images?

Reply:
Clarified